content/learn/book/ecs/_index.md

```diff
@@ -7,211 +7,76 @@ page_template = "book-section.html"
-
-Bevy's default execution strategy runs systems in parallel.
-Because the **function signature** of each of our systems fully define the data it can access, we can ensure that only one system can change a piece of data at once (although any number can read from a piece of data at the same time).
-
-Systems within the same **stage** are allowed to run in parallel with each other (as long as their data access does not conflict), and are assigned to a thread to perform work as soon as one is free.
-
-```rust
-fn main(){
-    let app = App::new()
-        .add_plugins(MinimalPlugins)
-        // Startup systems run exactly once, when the schedule is first run
-        .add_startup_system(spawn_damaged_player)
-        // Regular systems are run each time the schedule loops
-        .add_system(regenerate_life)
-        // We can use .before and .after to enforce a consistent ordering of our systems
-        .add_system(print_player_life.after(regenerate_life));
-
-    for i in 1..10 {
-        // This runs our app's schedule a single time
-        // When writing real games, you'll typically want to use App::run to loop the schedule indefinitely,
-        // but this method is very valuable for testing and teaching purposes!
-        app.update();
-    }
-}
-```
+In Bevy, game objects are stored as **entities**, whose data is stored as **components**.
+**Systems** operate on this data, modifying the **world** to carry out the behavior that brings your game to life.
+Together, these these form the basis of Bevy's **ECS**, which unsurprisingly stands for ["Entity-Component-System"](https://en.wikipedia.org/wiki/Entity_component_system).
+Let's go over the most important definitions:
+
+- **Entities:** Game objects (either abstract, like a camera, or tangible, like a player character), whose data is stored as components.
+  - The [`Entity`] type is just a simple identifier (like a URL address, a unique name, or a row number in a database): any combination of components can be added to each entity.
+- **Components:** Data stored on an entity, that can be manipulated in systems.
+  - Each component has a different Rust type that implements the [`Component`] trait, and only one component of each type can exist for each entity.
+- **World:** A unifying collection of all of the data stored in the ECS.
+- **Systems:** Special functions that read and write data stored in the [`World`].
+  - Any function whose parameters all implement the [`SystemParam`] type can be converted into a [`System`].
+
+Suppose we wanted to make a Breakout game in Bevy.
+Let's think about what entities we might want, what components they might have, and what systems we might create:
+
+- Paddle entities
+  - a dataless `Paddle` **marker component**, to allow us to uniquely identify the paddle
```

Comment:
Might be better to move the "marker component" clarification and definition up as one of the bullet points under `components`. Or maybe add a bullet here describing it, since right now it's not super clear that a "dataless" component _is_ a marker component.